べっちさん課題/index1.html

```diff
@@ -0,0 +1,33 @@
+!DOCTYPE html>
```

Comment:
`<` が抜けてしまっています。
そもそも `<!DOCTYPE html>` 自体があまり必要ないかと思います。

```suggestion
<!DOCTYPE html>
```